site/docs/theming/index.mdx

```diff
@@ -0,0 +1,113 @@
+
+## What a theme is
+
+A theme is a controlled change (or series of changes) to the design system's default design language. The goal is to change the system's look and feel to better fit with alternative established guidelines.
+
+## When to theme
+
+Salt offers a consistent design language ready for teams to use in their projects. Theming is not mandatory, but it can help a team meet their project's requirements.
+
+Before theming, consider these three questions:
+
+1. Is the default Salt design language unsuitable for my project?
+2. Do I have a set of alternative design guidelines to follow?
+3. Does my project need to follow those guidelines?
+
+If the answer to all three is "yes," theming may be suitable for your project.
+
+<ExampleContainer type="positive" customPillText="Best practice">
+  Use theming when your project needs to follow alternative design guidelines.
+</ExampleContainer>
+
+## How to theme
+
+Theming should aim to maintain a consistent look and feel. Salt is designed to ensure visual consistency, and understanding how Salt achieves this is key to producing consistent themes.
+
+While users can modify individual components to meet their needs, this can create inconsistencies as users have to remember to apply changes where relevant to other components. Salt's tokens make it easier to apply changes while maintaining consistency.
+
+### What a token is
+
+A token stores design decisions. Multiple components can use the same token to ensure a given design decision applies consistently across the design system.
+
+An example of a token is `--salt-status-error-foreground`. This token is used to set the foreground color when a component communicates an error message. That means both `Dialog` and `Toast` can use this token to set the foreground color when necessary.
+
+While a user could change the status foreground color for `Toast`, this wouldn't change the corresponding color for the `Dialog` component. Changing the color associated with the `--salt-status-error-foreground` token ensures that both components, and any others that refer to the token, all use the same color in the given context.
+
+<ExampleContainer type="positive" customPillText="Best practice">
+  When theming, we recommend modifying tokens rather than applying tweaks to
+  each individual component.
+</ExampleContainer>
+
+### Benefits of tokens
+
+Tokens offer several benefits:
+
+- Streamlined design changes that automatically apply to related components
+- Increased consistency across all components, improving the overall user experience
+- Simplified approach when adding new components or patterns
+- Single reference points for commonly-used values
+
+### Which tokens to modify
+
+Salt is under active development, and future versions may modify the token list. Some tokens, however, are less likely to change than others.
+
+Salt's tokens are organized into three tiers. From most generalized to most specific, the tiers are:
+
+1. Foundational tokens
```

Comment:
Can we add a brief description:

Foundation layer: The full ramps available to the theme.
Palette layer: A subset of foundation tokens defining the theme and used across characteristics.
Characteristic layer: Groups of tokens applied to components with common semantic meaning (is it a 'container', is it 'actionable', does it represent 'status').

@origami-z